src/OpenRoad.tcl

```diff
@@ -83,7 +83,7 @@ proc read_def { args } {
   set floorplan_init [info exists flags(-floorplan_initialize)]
   set incremental [info exists flags(-incremental)]
   if { $floorplan_init && $incremental } {
-    utl::error ORD 16 "incremental and floorplan_initialization options are both set. At most one should be used."
+    utl::error ORD 16 "Options -incremental and -floorplan_initialization are both set. Use one option."
```

Comment:
"Use one option" implies you can't omit both but "At most one should be used" implies both can be omitted.  I think we should stay with that portion of the original text.